tests/safety/test_ford.py

```diff
@@ -69,6 +80,11 @@ def setUp(self):
     self.safety.set_safety_hooks(Panda.SAFETY_FORD, 0)
     self.safety.init_tests()
 
+  def _curvature_meas_msg_array(self, curvature, speed):
```

Comment:
i'd call this something like reset_curvature_measurements

Reply:
just copied the convention for angle